app/abr-menu.js

```diff
@@ -30,6 +31,10 @@ function preprocessTemplate (element, config) {
             win.webContents.send("command", command, parameters);
         },
         replaceAttributes = function (item) {
+            if (item.label) {
+                item.label = localize.get(config, item.label);
```

Comment:
My guess is that using the item ID would prevent sharing a key between items. Am I right?

Maybe we should create a new property "key" to items in the JSON files. When localizing an item, we could:
* Check if it has a "key" property but misses a "label" property
* If so, add the "label" property set with the translation and remove the "key" property